react/package.json

```diff
@@ -15,7 +16,11 @@
   },
   "devDependencies": {
     "@vtex/test-tools": "^0.1.2",
-    "apollo-client": "^2.5.1"
+    "apollo-client": "^2.5.1",
+    "eslint": "^5.16.0",
+    "eslint-config-vtex-react": "^4.1.0",
+    "prettier": "^1.17.1",
+    "typescript": "^3.4.5"
```

Comment:
remove

Reply:
Actually, it is necessary. @lucasecdb knows better about it.

Reply:
If I'm not wrong it's a dependency to use eslint + prettier, even in JS projects.

Reply:
yep, obligatory